build/copyright/ccri-dstl-copyright-2017.txt

```diff
@@ -1,6 +0,0 @@
-Copyright (c) ${project.inceptionYear}-2016 ${owner}
-Portions Crown Copyright (c) 2017 Dstl
```

Comment:
oops, there were static `2017`s in the dstl files, replaced those with `${year}`